{{cookiecutter.project_dashed}}-src/ansible/group_vars/all

```diff
@@ -39,7 +41,7 @@ nginx_vhost_ssl_cert_key_fpath: /etc/letsencrypt/live/{{ nginx_vhost_server_name
 
 pyapp3_app_type: wsgi
 # If you change this, make sure you change the pip version used in wheelhouse.ini.
-pyapp3_python_version: 3.5
+pyapp3_python_version: 3.6
```

Comment:
3.7 or even 3.8 now.

Reply:
I wouldn't do 3.8 until it's actually released.